holopy/inference/tests/test_prior.py

```diff
@@ -46,7 +46,7 @@ def test_construction(self):
                       'upper_bound': 3,
                       'guess': 2,
                       'name': 'a'}
-        u = Uniform(*parameters.values())
+        u = Uniform(**parameters)
```

Comment:
my intention here (and on others below) was to verify the order of arguments, which is captured by * but not **. We already decided to only support python 3.7 in #254. We could also pass in matching lists of keys and values to check ordering without relying on an ordered dictionary.

Reply:
OK. I used an OrderedDict (just for readability) and changed the test back to use ```*parameters.values()```, in commit 744dfa6.